StructureDefinition/England-OperationOutcome.xml

```diff
@@ -17,7 +17,7 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="This profile defines the NHS England constraints and extensions on the UKCore FHIR resource [UKCore OperationOutcome](https://simplifier.net/hl7fhirukcorer4/ukcoreoperationoutcome)" />
+  <description value="This profile defines the NHS England constraints and extensions on the FHIR UK Core resource [UK Core OperationOutcome](https://simplifier.net/hl7fhirukcorer4/ukcoreoperationoutcome)" />
```

Comment:
Simplifier URL is incorrect (missing hyphen) - should be: https://simplifier.net/hl7fhirukcorer4/ukcore-operationoutcome